File: examples/processors/delete.py
from curses import reset_shell_mode
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../.env.production")

reclient = Retab()

reclient.processors.delete(
    processor_id="proc_F0FE8DFqyouQdZXDTWRg0",
)

File: examples/node/automations/endpoints/delete.js
import { config } from 'dotenv';
import { Retab } from '@retab/node';

// Load environment variables
config({ path: '../../../.env.production' });

const reclient = new Retab();

const automation = await reclient.processors.automations.endpoints.delete(
  "endp_DDw72RgWgIfKFXGaWXcGu"
);

console.log('Endpoint deleted successfully');